### ISSUE TITLE
Arduino Nano 33 IoT BSP exports v1 gpio module

### ISSUE THREAD

**Issue by Joeyh021** [atsamd-rs/atsamd#498]
The [blinky example for the board](https://github.com/atsamd-rs/atsamd/blob/master/boards/arduino_nano33iot/examples/blinky_basic.rs) gives a deprecation warning when defining the led pin, which seems to be because the bsp is exporting the v1 gpio module. The BSP uses the deprecated `define_pins` macro, which is the cause of this.

**Comment by bradleyharden** [comment]
@Joeyh021, yes, that's a Tier 2 BSP (see #433), so it will no longer track master and is pinned to 0.13. We're preparing to remove the gpio::v1 code entirely, so that deprecation notice serves to warn anyone who might want to upgrade the HAL version.

If you'd like, you could create a PR to update the BSP. If you do, we will update its pinned HAL version to 0.14 whenever it's released.
